browser/brave_content_browser_client.cc

```diff
@@ -138,6 +138,21 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/ftx/ftx_protocol_handler.h"
 #endif
 
+#if BUILDFLAG(BRAVE_ADS_ENABLED)
+#include "brave/browser/brave_ads/ads_service_factory.h"
```

Comment:
this factory is not behind a buildflag in gn

Reply:
buildflags in gn and in the source must always match

Reply:
Fixed, buildflag was removed here: https://github.com/brave/brave-core/pull/9769